<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>엑셀 명세서 금액 합산기</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>엑셀 명세서 금액 합산기</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_load_excel">
      <property name="text">
       <string>엑셀 파일 불러오기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_keyword">
      <property name="placeholderText">
       <string>검색할 가맹점 키워드를 입력하세요</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_calculate">
      <property name="text">
       <string>합산하기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_sum_result">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
